isaaclab.sh

```diff
@@ -22,34 +22,81 @@ export ISAACLAB_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && p
 # Helper functions
 #==
 
-# extract the python from isaacsim
-extract_python_exe() {
-    # Check if IsaacSim directory manually specified
-    # Note: for manually build isaacsim, this: _build/linux-x86_64/release
-    if [ ! -z ${ISAACSIM_PATH} ];
-    then
-        # Use local build
-        build_path=${ISAACSIM_PATH}
+# extract isaac sim path
+extract_isaacsim_path() {
+    # Check if we have pip package installed inside conda environment
+    if ! [[ -z "${CONDA_PREFIX}" ]]; then
+        # Use the python executable to get the path
+        local python_exe=${CONDA_PREFIX}/bin/python
+        # Retrieve the path importing isaac sim and getting the environment path
+        if [ $(${python_exe} -m pip list | grep -c 'isaacsim-rl') ]; then
+            local isaac_path=$(${python_exe} -c "import isaacsim; import os; print(os.environ['ISAAC_PATH'])")
+        else
+            # If package not installed, use an empty path for failure
+            local isaac_path=''
```

Comment:
does this mean conda won't work with isaac sim installed from the launcher?

Reply:
Oop good catch. But actually yeah, seems the new changes (to remove sourcing of isaacsim variables inside conda) will break conda anyway when installed with binaries. But not sure how to add those variables in the activation hook without first having the user install isaacsim (from pip or binaries)

Reply:
Fixed this now to deal with all the cases 😬

Reply:
thanks! looks good to me :)